client/webserver/site/src/js/app.js

```diff
@@ -240,63 +240,57 @@ export default class Application {
     this.popupTmpl = Doc.tmplElement(this.popupNotes, 'note')
     this.popupTmpl.remove()
     this.tooltip = idel(document.body, 'tooltip')
-    const pg = this.page = Doc.parsePage(this.header, [
-      'noteIndicator', 'noteList', 'noteTmpl', 'marketsMenuEntry',
-      'walletsMenuEntry', 'noteMenuEntry', 'loader', 'profileMenuEntry',
-      'profileIndicator', 'profileSignout', 'innerNoteIcon', 'innerProfileIcon',
```

Comment:
Nice.  This was error prone.  We've forgotten to add and remove them many times.  e.g. `profileIndicator` doesn't exist